Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2160,7 +2160,16 @@ def lhegensim2018ml(fragment,howMuch):
 
 steps['RECOCOSDRUN2']=merge([{'--conditions':'auto:run2_data','--era':'Run2_2016'},steps['RECOCOSD']])
 steps['RECOCOSDRUN3']=merge([{'--conditions':'auto:run3_data_prompt','--era':'Run3'},steps['RECOCOSD']])
```

Comment:
What is the reason to keep `RECOCOSDRUN3` as it is not used anywhere?
Why don't add customization to it instead of defining a new step.

Reply:
We might want to recycle it when we'll have cosmics rereco with offline conditions during CRUZET, etc